administration/src/bp-modules/self-service/CardSelfServiceForm.tsx

```diff
@@ -63,7 +63,7 @@ const CardSelfServiceForm = ({
     }
     if (!cardValid || dataPrivacyAccepted !== DataPrivacyAcceptingStatus.accepted) {
       appToaster?.show({
-        message: <FormErrorMessage style={{ color: 'white' }} errorMessage={t('atLeastOneInputIsInvalid')} />,
+        message: <FormAlert style={{ color: 'white' }} errorMessage={t('atLeastOneInputIsInvalid')} />,
```

Comment:
:upside_down_face:  i think we may create a separate ticket to create a `ToastAlertMessage` component, since it will never be null and the text color will be white and its not a "form alert"

Alternatively we could just set sth is like `isToast = true` and set a defined color to avoid inline styling